tests/csm/rest/test_csm_load.py

```diff
@@ -319,6 +332,87 @@ def test_44794(self):
         self.log.info("##### Test completed -  %s #####", test_case_name)
 
 
+    @pytest.mark.jmeter
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-45719')
+    def test_45719(self):
+        """
+        CSM load testing with Get User Capacity while running IOs in parallel
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+
```

Comment:
Make sure user listed are not present on the setup. If so delete them.

Reply:
setup_class have delete_csm_users() call, that is deleting all the users, expect predefined.
initially had added,  but it was redundant so removed.